addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AutomationPanel.java

```diff
@@ -712,49 +692,21 @@ private void disableJobAndTestButtons() {
         getRemoveTestButton().setEnabled(false);
     }
 
-    private JScrollPane getOutputScrollpane() {
-        if (outputScrollpane == null) {
-            outputScrollpane = new JScrollPane();
-            outputScrollpane.setViewportView(this.getOutputArea());
-        }
-        return outputScrollpane;
-    }
-
-    private JTextPane getOutputArea() {
-        if (outputArea == null) {
-            outputArea = new JTextPane();
-            outputArea.setEditable(false);
-            styleError = this.getOutputArea().addStyle("Error", null);
-            StyleConstants.setIcon(styleError, RED_BALL_ICON);
-
-            styleWarning = this.getOutputArea().addStyle("Warning", null);
-            StyleConstants.setIcon(styleWarning, ORANGE_BALL_ICON);
-
-            styleInfo = this.getOutputArea().addStyle("Info", null);
-            StyleConstants.setIcon(styleInfo, WHITE_BALL_ICON);
-        }
-        return outputArea;
-    }
-
     private PlanTreeTableModel getTreeModel() {
         if (treeModel == null) {
             treeModel = new PlanTreeTableModel(new DefaultMutableTreeNode("Plan"));
         }
         return treeModel;
     }
 
-    private static String listToStr(List<String> list) {
-        return String.join("\n", list);
-    }
-
-    private void outputMessage(String message, Style style) {
-        StyledDocument doc = this.getOutputArea().getStyledDocument();
-        try {
-            doc.insertString(doc.getLength(), " ", style);
-            doc.insertString(doc.getLength(), "  " + message + "\n", null);
-        } catch (BadLocationException e) {
-            // Ignore
-        }
+    private void outputMessage(String message, Level level) {
+        String planName =
+                currentPlan.getFile() != null
```

Comment:
Getting NPEs on (re)installation:
```
Exception in thread "AWT-EventQueue-0"
java.lang.NullPointerException: Cannot invoke "org.zaproxy.addon.automation.AutomationPlan.getFile()" because "this.currentPlan" is null
	at org.zaproxy.addon.automation.gui.AutomationPanel.outputMessage(AutomationPanel.java:704)
	at org.zaproxy.addon.automation.gui.AutomationPanel.handleEvent(AutomationPanel.java:790)
	at org.zaproxy.addon.automation.gui.AutomationPanel.lambda$eventReceived$15(AutomationPanel.java:764)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:323)
```

Maybe related to https://github.com/zaproxy/zap-extensions/pull/6992/files#r2617079773 which was outputting after the plan was set?